google-cloud-pub-sub-grpc/src/main/resources/reference.conf

```diff
@@ -0,0 +1,12 @@
+alpakka.google.cloud.pubsub.grpc {
+  host = "pubsub.googleapis.com"
+  port = 443
+
+  # Set to "none" to disable TLS
+  rootCa = "GoogleInternetAuthorityG3.crt"
```

Comment:
Wouldn't it be better to put the certificate in a sub-directory, so that it works well even with über-jars?

Reply:
Why would having the certificate in the top directory not work with uber-jars?

Reply:
It works. Just thought it might be good to show context to where it is used. Maybe irrelevant.